wordstats/language_data/t2l/code/translations.py

```diff
@@ -0,0 +1,123 @@
+from query import query_site
+from pathlib import Path
+import time
+import math
+import random
+
+"""
+
+Class that calls a translation API to translate words and stores the
+translations as a txt file
+
+"""
+
+# First word from which to start translating
+BEGIN = 0
+
+# How many words to translate before storing them (the last one is not included)
+BATCH = 700
+
+FROM_CODE = "nl"
+DEST_CODE = "en"
+
+# Path of the txt file with the words
+PATH_INPUT_WORDS = str(Path(__file__).parent.parent) + '/data/words/' + FROM_CODE + '.txt'
+
+# Path of the txt file with the translations
+PATH_OUTPUT_TRANSLATIONS = str(Path(__file__).parent.parent) + "/data/translations/" + FROM_CODE + "-" + \
```

Comment:
result of running this: 

```
$ python translations.py 
Traceback (most recent call last):
  File "translations.py", line 123, in <module>
    main()
  File "translations.py", line 90, in main
    file = open(PATH_INPUT_WORDS)
FileNotFoundError: [Errno 2] No such file or directory: './data/words/fr.txt'

```

It works if I replace the Path(__file__).parent.parent. with `".."`. didn't we do this together at some point?